docs/pages/api/app-bar.md

```diff
@@ -6,7 +6,7 @@ filename: /packages/material-ui/src/AppBar/AppBar.js
 
 # AppBar API
 
-<p class="description">The API documentation of the AppBar React component. Learn more about the properties and the CSS customization points.</p>
+<p class="description">The API documentation of the AppBar React component. Learn more about the props and the CSS customization points.</p>
```

Comment:
I’m loving the slow transition 😁 It’s always great to see changes like this.

Reply:
Well, I'm not feeling very productive when doing that. I want to make sure people starting with us, use the right terminology :).

Reply:
I think you should, we’ve decided we want to change and you’re changing it. Terminology is important.